Python_Module/MangDang/mini_pupper/display.py

```diff
@@ -8,9 +8,20 @@
 def show_image(image):
 
     # init st7789 device
-    disp = ST7789()
+    disp = ST7789(
+        port=0, 
+        cs=0,
+        dc=24, 
+        backlight=17,
+        rst=27,
+        width=320,
+        height=240,
+        rotation=180,
+        spi_speed_hz=60 * 1000 * 1000,
+        offset_left=0,
+        offset_top=0
+   )
```

Comment:
```suggestion
    disp = ST7789()
```

Reply:
Thanks for your comment.
1. I tested w/ and w/o disp.clear() , I have not found the difference.
2. If change ST7789.py but not display.py, do we need to add some comment info at the head of ST7789.py?